cmd/deploy/deploy.go

```diff
@@ -86,7 +86,7 @@ func (d *deployCommand) PreRunE(_ *cobra.Command, _ []string) error {
 }
 
 func (d *deployCommand) RunE(_ *cobra.Command, _ []string) error {
-	d.logger.Info("Registering project [%s] to [%s]", d.clientConfig.Project.Name, d.clientConfig.Host)
+	d.logger.Info(fmt.Sprintf("Registering project [%s] to [%s]", d.clientConfig.Project.Name, d.clientConfig.Host))
```

Comment:
Let's not use `fmt.Sprintf` here since doing that defeats the purpose of the previous issue for this update, mentioned [here](https://github.com/odpf/optimus/issues/335). The issue's goal is to remove this `fmt.Sprintf` or the like entirely. Meaning, we aim it to be:
```go
d.logger.Info("Registering project [%s] to [%s]", d.clientConfig.Project.Name, d.clientConfig.Host)
```
not
```go
d.logger.Info(fmt.Sprintf("Registering project [%s] to [%s]", d.clientConfig.Project.Name, d.clientConfig.Host))
```

Reply:
updated